### PR TITLE
Geode-5396

### PR DESCRIPTION
**Pull request by mhansonp**
See pull request 2136 for previous pass of comments. Renamed branch due to naming conflict.

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by mhansonp** [comment]
@upthewaterspout @mcmellawatt @WireBaron @PurelyApplied Please review...

File: geode-pulse/src/test/java/org/apache/geode/tools/pulse/tests/Server.java [hunk 1]
```diff
@@ -99,105 +128,80 @@ public Server(int jmxPort, String properties, String jsonAuthFile) throws Except
       System.out.println("RMI registry ready.");
     } catch (Exception e) {
       System.out.println("Exception starting RMI registry:");
-      e.printStackTrace();
+      throw e;
     }
```
Comment (mcmellawatt) [inline geode-pulse/src/test/java/org/apache/geode/tools/pulse/tests/Server.java]: No need to catch if you are just going to rethrow

Comment (mhansonp) [reply geode-pulse/src/test/java/org/apache/geode/tools/pulse/tests/Server.java]: Good suggestion.

**Comment by mcmellawatt** [comment]
Can you rename this PR to be the GEODE-#: Description format?

**Comment by mhansonp** [comment]
@upthewaterspout @jinmeiliao  Please review this PR when you get the chance.

**Comment by mhansonp** [comment]
@upthewaterspout @WireBaron @jinmeiliao @mcmellawatt  Can one of you merge this? I am not a committer.

**Pull request merged**